devel/simple-web-server/Portfile

```diff
@@ -1,31 +1,31 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
 
 PortSystem          1.0
 PortGroup cmake     1.1
-PortGroup boost     1.0
```

Comment:
the `boost` PG is a recent improvement in MacPorts - ideally we don't want to revert back to the old way of specifying boost dependencies. Did you run into trouble with the current approach? If so, feel free to post the error message and we can try and figure out a solution.